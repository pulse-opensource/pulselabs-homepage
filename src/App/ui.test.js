const React = require('react');
const App = require('./').default;
const { shallow } = require('enzyme');
const { shallowToJson } = require('enzyme-to-json');

describe('AppUI', () => {
	it('renders the expected UI', () => {
		const wrapper = shallow(<App />);

		expect(
			shallowToJson(wrapper)
		).toMatchSnapshot();
	});
});
