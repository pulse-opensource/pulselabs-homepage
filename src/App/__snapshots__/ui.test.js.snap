exports[`AppUI renders the expected UI 1`] = `
<div>
  <span>
    <i>
      PULSE-labz
    </i>
  </span>
</div>
`;
